dashboard/test/factories/factories.rb

```diff
@@ -1391,6 +1391,13 @@
     comment 'a comment about your project'
   end
 
+  factory :reviewable_project do
+    sequence(:storage_app_id)
+    association :user, factory: :student
+    association :level
+    association :script
+  end
```

Comment:
Ah nice, I added this in my PR as well - for my knowledge, what is `sequence(:storage_app_id)` needed for?

Reply:
I believe it's so that if you created multiple reviewable projects in a test, they would have unique storage app IDs -- more details here:

https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences

Reply:
I did just realize though that my factory does not work on its own (it works in the tests, b/c I provide user/script/level IDs) unless you've explicitly added the level/script/user associations in the `ReviewableProject` model. Added those so the factory will work if anyone uses it in the future.

Reply:
Gotcha, yep `ReviewableProject` should have those associations now